client/post-editor/media-modal/secondary-actions.jsx

```diff
@@ -62,6 +64,11 @@ const MediaModalSecondaryActions = React.createClass( {
 		this.props.onChangeView( ModalViews.DETAIL );
 	},
 
+	navigateToPlans() {
+		analytics.ga.recordEvent( 'Media', 'Clicked Plan Storage Button' );
+		page( `/plans/${ this.props.site.slug }` );
```

Comment:
I think it'd be safe enough to do so unless our site loading strategy changes in the future. By the time you get to the media modal the site in the sites tree should be loaded.